pkg/osquery/table/onepassword_config.go

```diff
@@ -0,0 +1,83 @@
+package table
+
+import (
+	"context"
+	"database/sql"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+
+	"github.com/kolide/kit/fs"
+	"github.com/kolide/osquery-go"
+	"github.com/kolide/osquery-go/plugin/table"
+)
+
+type onePasswordAccountConfig struct {
+	client *osquery.ExtensionManagerClient
+}
+
+// OnePasswordAccountConfigGenerate will be called whenever the table is queried. It should return
+// a full table scan.
+func (o *onePasswordAccountConfig) generate(ctx context.Context, queryContext table.QueryContext, results []map[string]string) ([]map[string]string, error) {
```

Comment:
I'm a little uneasy with the API here. I'd prefer to see it be
```
func (o *onePasswordAccountConfig) generate(ctx context.Context, queryContext table.QueryContext) ([]map[string]string, error)
```
and then do the append at the top level.